src/pages/stacks-blockchain/technical-specs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Technical Specifications
+description: Summary of technical specifications of Stacks 2.0
+---
+
+## Consensus
+
+- Proof of Transfer (PoX) as described in [SIP-007](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-007-stacking-consensus.md)
+- Network will transition to Proof of Burn (PoB) as described in [SIP-001](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md) after 10 years. More details [here](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md).
+
+## Proof of Transfer Mining
+
+- Coinbase reward schedule:
+  - 1000 STX/block for first 4 years
+  - 500 STX/block for following 4 years
+  - 250 STX/block for subsequent 4 years
+  - 125 STX/block in perpetuity after that
+- Reward maturity window: 100 blocks, meaning leaders will earn the coinbase reward 100 blocks after the block they successfully mine.
+- Block interval: Stacks blockchain produces blocks at the same rate as the underlying burnchain. For Bitcoin, this is approximately every 10 minutes.
+
+## Stacking
+
+- Stacking works in 2 phases
+  1. Prepare phase: In this phase an "anchor block" is chosen. The qualifying set of addresses ("reward set") is determined based on the snapshot of the chain at the anchor block. Length of prepare phase is 250 blocks (overlaps with reward cycle of the previous phase).
+  2. Reward phase: In this phase miner BTC commitments are distributed amongst the reward set. Reward cycle length is 2000 BTC blocks (~2 weeks).
+- Two reward addresses / block, for a total of 4000 addresses every reward cycle. The addresses are chosen using a VRF (verifiable random function), so each node can deterministically arrive at the same reward addresses for a given block.
+- Stacking threshold: 0.025% of the participating amount of STX when participation is between 25% and 100% and when participation is below 25%, the threshold level is always 0.00625 of the liquid supply of STX.
+- Delegation: An STX address can designate another address to participate in Stacking on its behalf.
```

Comment:
maybe link to the SIP description for delegation (while we don't have good delegation docs)?

https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-007-stacking-consensus.md#stacker-delegation